test_runner/fixtures/neon_fixtures.py

```diff
@@ -3806,6 +3805,19 @@ def create(
         # and make tests more stable.
         config_lines = ["max_replication_write_lag=15MB"] + config_lines
 
+        # Delete file cache if it exists (and we're recreating the endpoint)
+        if (lfc_path := Path(self.lfc_path())).exists():
+            lfc_path.unlink()
+        else:
+            lfc_path.parent.mkdir(parents=True)
```

Comment:
Weird how this doesn't suddenly work with the smaller cases: https://neon-github-public-dev.s3.amazonaws.com/reports/pr-8613/10270741093/index.html#suites/140824de6e814b5b1ae2b622c3f67840/5412e1bdfaac3f6a